equinoxfitness/scripty
# Datplat 1711 add default from date

## Description

get default from date from a global variable or system environment variable

## Changes

- Add default variable to init.py file
- System env DEFAULT_FROM_DATE is also applicable to be used to change default from date.

### How Has This Been Tested?


_Example:_
1. Checkout this branch

1. Switch to virtual environment

1. Run `python -m scripty.module -c core -s 'sample/script_runner_test.sql' -f '2020-03-05'`
- [ ] Expect 2020-03-05 in the console.
1. Run `python -m scripty.module -c core -s 'sample/script_runner_test.sql'`
- [ ] Expect 1776-07-04 in the console.
1. Run `export DEFAULT_FROM_DATE=1999-06-04`
1. Run `python -m scripty.module -c core -s 'sample/script_runner_test.sql'`
- [ ] Expect 1999-06-04 in the console.



### Rollback Plan

What to do if something goes wrong?
